routes/api.js

```diff
@@ -70,6 +81,47 @@ router
     }
 
     res.json({ authenticated: !!user });
+  })
+  .post('/no_shows', isUserAuthenticated, async (req, res, next) => {
+    try {
+      const { access_token, refresh_token } = req.user;
+      const { invitee } = req.body;
+
+      const calendlyService = new CalendlyService(access_token, refresh_token);
+
+      const { resource } = await calendlyService.markAsNoShow(invitee);
+
+      res.json({ resource });
+    } catch (error) {
+      next(error);
+    }
+  })
+  .delete('/no_shows/:uuid', isUserAuthenticated, async (req, res, next) => {
+    try {
+      const { access_token, refresh_token } = req.user;
+      const { uuid } = req.params;
+      const calendlyService = new CalendlyService(access_token, refresh_token);
+
+      await calendlyService.undoNoShow(uuid);
+
+      res.status(204).send('No-show successfully undone');
```

Comment:
I think we can just send an empty body response here. So you should be able to replace this with:

```js
res.status(204).end()
```